test_launch_testing/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>test_launch_testing</name>
+  <version>0.7.3</version>
+  <description>Tests for the ROS launch testing suite.</description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
```

Comment:
Did this package already exist? I don't think so correct? If it did not, let's remove the author's and set the maintainer to me and/or you @hidmic and/or @pbaughman, if you guys want it. If not, then you guys should be in the author's list.

Reply:
I've put @ivanpauno and myself as authors, and myself as maintainer. That's as accurate as possible.

Reply:
Fine by me. 👍